readme.md

```diff
@@ -189,6 +189,7 @@
 	- [Protips](https://github.com/AllThingsSmitty/css-protips)
 - [React](https://github.com/enaqx/awesome-react) - App framework.
 	- [Relay](https://github.com/expede/awesome-relay) - Framework for building data-driven React apps.
+	- [React Bootstrap Components](https://github.com/Hermanya/awesome-react-bootstrap-components) - Extra components for [react-bootstrap](https://react-bootstrap.github.io/)
```

Comment:
I don't think descriptions should have a hyperlink, though this is not mentioned in the guidelines.